doc/build-versioned-docs.sh

```diff
@@ -49,18 +49,36 @@ for current_version in ${versions}; do
       # Copy updated version of locales
       cp -r ${tmp_dir}/locales/$current_language locales/
       changed=true
-
     fi
 
+    # Only for v1.5.0, update the versions listed in the switcher
```

Comment:
@charlesbvll does this mean this code segment will not be applied in 1.5.1 and subsequent releases?

Reply:
Yes, because for versions above 1.5.0, the correct code is already written in `conf.py`.

Reply:
Sounds good! merging!